api/middleware/responseLogger_test.go

```diff
@@ -25,7 +25,7 @@ func startApiServerResponseLogger(handler groups.AccountsFacadeHandler, respLogM
 	accGr, _ := groups.NewAccountsGroup(handler)
 
 	group := ws.Group("/address")
-	accGr.RegisterRoutes(group, data.ApiRoutesConfig{}, func(_ *gin.Context) {}, func(_ *gin.Context) {})
+	accGr.RegisterRoutes(group, data.ApiRoutesConfig{}, func(_ *gin.Context) {}, func(_ *gin.Context) {}, func(_ *gin.Context) {})
```

Comment:
same here and in api/middleware/rateLimiter_test.go regarding the same pointer function

Reply:
done